File: problemSolvingPY/positivenegativecounter.py
posCount = 0
negCount = 0
while True:
    number =int(input("Input the next number"))
    if(number > 0):
        posCount += 1
    if(0 > number):
        negCount += 1
    if(number == 0):
        break;

print("Positive Numbers:" + str(posCount) + "   Negative Numbers:" + str(negCount))

File: problemSolvingC/positivenegativecounter.c
#include <stdio.h>
int main(){
	int posCount = 0;
	int negCount = 0;
	int number;
	do {
		printf("Input the next number\n");
		scanf("%i", &number);
		if(number > 0){
			posCount = posCount + 1;
		}
		if(number < 0){
			negCount = negCount + 1;
		}
	} while( number != 0);
	printf("Positive Numbers: %i    Negative Numbers %i\n", posCount, negCount);
}
